tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot

```diff
@@ -26,6 +25,14 @@ Verify Pachyderm Can Be Installed Using OpenShift Console
     Wait Until Status Is Running
     Go To RHODS Dashboard
     Verify Service Is Enabled    Pachyderm
+    Launch JupyterHub Spawner From Dashboard
+    Wait Until Page Contains Element    xpath://input[@name="Standard Data Science"]
+    Wait Until Element Is Enabled       xpath://input[@name="Standard Data Science"]    timeout=10
+    Spawn Notebook With Arguments  image=s2i-generic-data-science-notebook
+    Create Pachyderm Pipeline Using JupyterLab
+    ${status}=    Check If POD Exists    pachyderm      app=pipeline-edges-v1
```

Comment:
I think it is necessary to check the pod status, it should be "running" and the TC may need to wait until the pod gets that status

Reply:
Done !